spec/fixtures/exports/case_logs.xml

```diff
@@ -158,6 +158,23 @@
     <previous_postcode_known>1</previous_postcode_known>
     <previous_la_known/>
     <is_previous_la_inferred>false</is_previous_la_inferred>
+    <age1_known/>
```

Comment:
I don't think we want to be exporting any of these since they're just specific to the form UI not the other entry methods.

Reply:
OK I think I saw some errors from the the tests/linting but I think they might have been around diff I can remove these

Reply:
@baarkerlounger Is it all of the new fields that need omitting of just the **_known** fields

Reply:
I think just the `_known` fields.

Reply:
Do I need to amend  the export service to omit these fields then?

**app/services/exports/case_log_export_service.rb**

Reply:
Yes I think so

Reply:
amended